### ISSUE TITLE
Is marketplacekit still active?

### ISSUE THREAD

**Issue by jbxbergdev** [marketplacekit/marketplacekit#130]
Considering the last commit was last October, I'm wondering whether the company behind Marketplacekit is still actively developing the software. Can anybody share some insight?

**Comment by Ezyweb-uk** [comment]
I contacted support in May and they said they had recently launched a new version of their hosted option, and that they were planning to update the self-hosted version by the end of June. So I contacted them again end of June and they said they were planning for end of July. End of July has now gone and we are into August. Seems like they are concentrating on their hosted version, which considering the amount of interest on GitHub will likely limit development progress.

**Comment by faierbol** [comment]
Hi Guys, i have also sent them an email, hopefully they will push some updates. 
I am also planning to use this app for my next project but currently has some bugs.

**Comment by faierbol** [comment]
The response was that the updates will be probably pushed by the end of September, but this is not a fixed date.

**Comment by PwrSrg** [comment]
I am pretty sure they have little to no interest or intention of updating the self-hosted version since they released the hosted platform, and are simply trying to push everyone to that instead.

I asked them this question back in **MARCH**, and this was their response:

```
Hi Sergio,

Thanks for your interest in MarketplaceKit. But unfortunately, we've stopped custom dev and selling add-ons for the self-hosted version for some time since the self-hosted version of the platform is not up-to-date. We've just released a newer version of the platform on the hosted version which features all the features and add-ons, and we've yet to publish them to the open-source version.

We are planning to release the new version to GitHub soon, but have not finalized the date yet. You can, however, try out all the new features and add-ons on the hosted version of MarketplaceKit by creating an account on https://marketplacekit.com/

Regards,
Sean

Customer Support
MarketPlaceKit
```

Very disappointing.

As a result, I ended up going with **LaraShip** which is a fully-polished, full-featured marketplace platform.

**LaraShip** - https://www.laraship.com/product/laraship-laravel-marketplace/
	Licenses: Standard - $100 for 6mo updates, Extended - $160 for 1yr updates, MultiTenant - $500 for 1yr updates plus unlimited domains and subdomains
	They do not offer any kind of Trial or Money Back Guarantee.
	WHY LaraShip - https://github.com/saeed-corals/laravel-admin
	Demos - https://www.laraship.com/laraship-demo/
	Docs - https://www.laraship.com/docs/laraship/welcome-to-laraship/


Here are also a few others seriously worth considering:

**ZigKart** - https://codecanyon.net/item/zigkart-multivendor-products-marketplace/25801356
	$50 - Multivendor Products Marketplace
	Demo - https://codecanor.com/demo/zigkart
	Admin - https://codecanor.com/demo/zigkart/login
	Docs - https://codecanor.com/docs/zigkart/documentation/

**GeniusCart** - https://codecanyon.net/item/geniuscart-single-or-multivendor-ecommerce-system-with-physical-and-digital-product-marketplace/24089099
	$70 - Single or Multivendor Ecommerce System with Physical and Digital Product Marketplace
	Demo - https://geniusocean.com/demo/geniuscart/default/
	Website link : http://geniuscart.local
	Admin link : http://geniuscart.local/admin
	ALL Passwords: 1234

**KingCommerce** - https://codecanyon.net/item/kingcommerce-all-in-one-singlemulti-vendor-ecommerce-business-management-system/22716806
	$135 - All in One Single and Multivendor Eommerce Business Management System
	NOT sure what the difference is between this and GeniusCart.  GeniusCart actually looks much nicer and newer for LESS.

**Bagisto** - https://bagisto.com/en/laravel-multi-vendor-marketplace/
	$350 - Super LEGIT "FREE" open source eCommerce platform, but the "marketplace" plugin is $350.
		https://store.webkul.com/laravel-multi-vendor-marketplace.html
	Demo - https://demo.bagisto.com/marketplace/
	Docs - https://devdocs.bagisto.com
	Blog - https://bagisto.com/en/blog/

**zCart** - https://codecanyon.net/item/zcart-multivendor-ecommerce-platform/22751042
	$300 - Multi-Vendor eCommerce Marketplace
	Actually one of the nicer Seller/Merchant interfaces of all of them.  Still expensive and kinda dated.
	Docs - https://support.incevio.com/docs/
	Storefront Demo - https://zcart.incevio.com
	Merchant Landing Page Demo - https://zcart.incevio.com/selling
	Admin Demo - https://zcart.incevio.com/login
	Customer Account Demo - https://zcart.incevio.com/customer/login

**Comment by sanskritkm** [comment]
You can try this Repo instead. https://github.com/wcmarketplace/dc-woocommerce-multi-vendor

**Comment by Ezyweb-uk** [comment]
One I've been looking at is https://github.com/kirilkirkov/Ecommerce-CodeIgniter-Bootstrap

**Comment by PwrSrg** [comment]
> You can try this Repo instead. https://github.com/wcmarketplace/dc-woocommerce-multi-vendor

1. I would never build a Multi-Vendor Marketplace on Wordpress.
2. If I HAD to, there are LOTS of better plugins on the market that are not riddled with bugs like WC Marketplace.

    Here are the top 3:

    - Dokan Multi-vendor - https://wedevs.com/dokan
    - WC Vendors - Multi Seller Marketplace - https://www.wcvendors.com
    - WCFM Marketplace - https://wclovers.com

**Comment by PwrSrg** [comment]
> One I've been looking at is https://github.com/kirilkirkov/Ecommerce-CodeIgniter-Bootstrap

This looks interesting, but in 2020 I would ONLY consider using Codeigniter if it were built on v4.

**Comment by Nuranto** [comment]
Sad to see the only open source free-to-use marketplace framework is already dead....
Do you know other free solution ? I'm on a side-work POC project on which I don't want to invest anything but time until it proves itself.... If I had a budget, I would probably have gone with aimeos.. 

I saw [Sellacious](https://www.sellacious.com/) seems to be free, but I'm not fond of Joomla...
I skip all WP plugins too...

I'm really seduced by a headless approach, the only one I found is [Shuup](https://www.shuup.com/)... 

Do you know others ?

**Comment by izdrail** [comment]
Yeah, I had the same question and saw the project is dead :) I guess somebody should fork it and actually do some work on it

**Comment by JP-FStreet** [comment]
Hey guys - I'm looking for a hosted alternative. Do you have any recommendations? I see @PwrSrg mentioned a few, how is Lara ship going?

**Comment by PwrSrg** [comment]
"_A FEW_"??  **LOL**  You can't **get** a better, more detailed recomendation list than what I provided just seven months ago!  😆

But good luck!

**Comment by JP-FStreet** [comment]
@PwrSrg haha ok ok, credit where credit is due! How is Laraship going? Happy with the choice?

**Comment by PwrSrg** [comment]
> @PwrSrg haha ok ok, credit where credit is due! How is Laraship going? Happy with the choice?

Well, Thank You!  😁

So funny story...  After buying into a 2-year **LaraShip** license, working on it for **over 6 months**, and going through **5 developers**, I ended up **bailing** on it, and building the **entire thing from scratch in CodeIgniter 4**.  I had been following the **Ci4 Alpha build** for over 2 years, and when it was suddenly released, I happened to be in the middle of a LaraShip **MESS**.  _NOT_ because of the software, LaraShip is great!  But the project just got off to a bad start, and ran into a string of bad luck.  So the timing for Ci4 was **perfect**, and could not have come any sooner!

Anyways, if you are in the market for an **all-in-one Marketplace Package**, then everything I outlined, and all of the details, **still apply**!  There is not a huge market for this, so your options are a _bit limited_.  But **YES**, I would _STILL_ recommend **LaraShip** over all other options available right now.

Take care and Good Luck!  👍🏻

**Comment by Ezyweb-uk** [comment]
I'm still looking and recently found Modesy on Envato for $69. Seems to be feature rich but currently it is CodeIgniter 3 rather than 4.

**Comment by ugintl** [comment]
Still no open source solution available?
